ext/nnstreamer/tensor_filter/vivante/README.md

```diff
@@ -0,0 +1,166 @@
+
+This describes how to enable the NNstreamer tensor filter as a sub-plugin for Vivante NPU.
+
+## Vivante tensor filter
+The entire procedure is as follows:
+ * Read multi-files (e.g., vivante_v3.nb and libvivante_v3.so)
+ * Create a neural network
+ * Execute pre-process for the image data
+ * Verity a graph
+ * Process the graph
+ * Dump all node outputs (optionally, for a debugging)
+ * Do post-process for output data
+ * Release the neural network
+
+## Reference
+ * http://www.vivantecorp.com/
+ * https://www.khadas.com/product-page/vim3 (Amlogic A311D with 5.0 TOPS NPU)
+ 
+## How to buid
+First of all, you must generate a library of a model (e.g., libvivantev3.so, libyolov3.so) to use NNStreamer tensor filter.
+For more details, please refer to the below repository.
+ * https://www.github.com/nnstreamer/vivante-nnstreamer-filter (TODO) - Press 'models' folder.
+
+
+### on VIM3/Ubuntu18.04 (arm64)
+Please append the APT repository as follows for VIM3/Ubuntu 18.04 target board.
+```bash
+vim3$ sudo /usr/sbin/ntpdate jp.pool.ntp.org
+vim3$ sudo apt-get update -o Acquire::https::dl.khadas.com::Verify-Peer=false
+vim3$ cat /etc/apt/sources.list.d/fenix.list
+deb [trusted=yes] https://dl.khadas.com/repos/vim3/ bionic main
+```
+
+Build and run the NNStreamer tensor filter on Ubuntu18.04 (arm64) + the VIM3/Khadas board.
+
+```bash
+vim3$ git clone https://github.com/nnstreamer/nnstreamer.git
+vim3$ cd ./nnstreamer/ext/nnstreamer/tensor_filter/vivante/
+vim3$ ./build_run.sh
+```
+
+### on Tizen/Unified (armv7l)
+Build and run the NNStreamer tensor filter on Tizen (armv7l).
+
+```bash
+$ cp ./gbs-build-conf/.gbs.conf ~/
+$ vi ~/.gbs.conf
+$ gbs build -A armv7l --clean --include-all
+```
+
+## How to run
+You can get the below result as follows when the required packages are prepared on the target board.
```

Comment:
IMHO, 'the below result' and 'as follows' could be a duplicate phrase.

Reply:
Fixed. Thanks.